app/models/manageiq/providers/embedded_ansible/automation_manager/machine_credential.rb

```diff
@@ -44,20 +44,7 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::MachineCredential
       :id               => 'become_method',
       :type             => 'choice',
       :isClearable      => true,
-      :options          => %w[
-        sudo
-        su
-        pbrum
-        pfexec
-        doas
-        dzdo
-        pmrun
-        runas
-        enable
-        ksu
-        sesu
-        machinectl
-      ].map { |item| {:label => item, :value => item} },
+      :options          => ['sudo', 'su', 'pbrun', 'pfexec', 'doas', 'dzdo', 'pmrun', 'runas', 'enable', 'ksu', 'sesu', 'machinectl']
```

Comment:
@GilbertCherrie can you help here?  My understanding was `:options` should be formatted as an array of `{:label =>, :value =>}` hashes

Reply:
@kTipSSIoYv  @agrare I'm not too sure but would another possible fix be to keep the options as is but just change to values to be strings, so just add ' ' around each item?

Reply:
Awesome thanks @GilbertCherrie !

Reply:
Tested this locally and it is working fine:
<img width="1414" alt="Screen Shot 2022-06-06 at 10 04 22 AM" src="https://user-images.githubusercontent.com/32444791/172176577-0667dec8-55ba-4696-8033-1359da706f5e.png">